cli/tests/integration/repl_tests.rs

```diff
@@ -623,6 +623,52 @@ fn missing_deno_dir() {
   assert!(err.is_empty());
 }
 
+#[test]
+fn custom_history_path() {
+  use std::fs::read;
+  let temp_dir = TempDir::new();
+  let history_path = temp_dir.path().join("history.txt");
+  let (out, err) = util::run_and_collect_output(
+    true,
+    "repl",
+    Some(vec!["1"]),
+    Some(vec![
+      (
+        "DENO_REPL_HISTORY".to_owned(),
+        history_path.to_str().unwrap().to_owned(),
+      ),
+      ("NO_COLOR".to_owned(), "1".to_owned()),
+    ]),
+    false,
+  );
+  assert!(read(&history_path).is_ok());
+  assert_ends_with!(out, "1\n");
+  assert!(err.is_empty());
+}
+
+#[test]
+fn disable_history_file() {
+  let deno_dir = util::new_deno_dir();
+  let default_history_path = deno_dir.path().join("deno_history.txt");
+  let (out, err) = util::run_and_collect_output(
+    true,
+    "repl",
+    Some(vec!["1"]),
+    Some(vec![
+      (
+        "DENO_DIR".to_owned(),
+        deno_dir.path().to_str().unwrap().to_owned(),
+      ),
+      ("DENO_REPL_HISTORY".to_owned(), "".to_owned()),
+      ("NO_COLOR".to_owned(), "1".to_owned()),
+    ]),
+    false,
+  );
+  assert!(!default_history_path.try_exists().unwrap());
```

Comment:
I think this can just be the following?

```suggestion
  assert!(!default_history_path.exists());
```

Reply:
`Path::exists` returns false when an error occurs while trying to access the file's metadata, for example on a permission error from the parent directory, even if the file does in fact exist, resulting in a falsely passing test. In practice this should never happen but it's easy to avoid the possibility entirely.

Reply:
Interesting. I didn’t not know that.